statistics: add some doc for the exp feature (#9891) (#10534) (#10537)
pingcap/docs: extended-statistics.md

TiDB can collect the following two types of statistics:

> Basics: statistics such as histograms and Count-Min Sketch. See [Introduction to Statistics](/statistics.md) for details.

Reviewer: ```suggestion
- Basic statistics: statistics such as histograms and Count-Min Sketch. See [Introduction to Statistics](/statistics.md) for details.
```

TiDB can collect the following two types of statistics:

> <<<<<<< HEAD
> - Regular statistics: statistics such as histograms and Count-Min Sketch. See [Introduction to Statistics](/statistics.md) for details.
> =======
> - Basic statistics: statistics such as histograms and Count-Min Sketch. See [Introduction to Statistics](/statistics.md) for details.
> >>>>>>> b270933fe (statistics: add some doc for the exp feature (#9891) (#10534))

Reviewer: ```suggestion
Basics: statistics such as histograms and Count-Min Sketch. See [Introduction to Statistics](/statistics.md) for details.
```

### Export and import extended statistics

> <<<<<<< HEAD
> The way of exporting or importing extended statistics is the same as exporting or importing regular statistics. See [Introduction to Statistics - Import and export statistics](/statistics.md#import-and-export-statistics) for details.
> =======
> The way of exporting or importing extended statistics is the same as exporting or importing basic statistics. See [Introduction to Statistics - Import and export statistics](/statistics.md#import-and-export-statistics) for details.
> >>>>>>> b270933fe (statistics: add some doc for the exp feature (#9891) (#10534))

Reviewer: ```suggestion
The way of exporting or importing extended statistics is the same as exporting or importing basic statistics. See [Introduction to Statistics - Import and export statistics](/statistics.md#import-and-export-statistics) for details.
```
